src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -139,6 +142,53 @@ addWatchFile('/home/user/.../my-config.json');
 addWatchFile(new URL('./tailwind.config.js', config.root));
 ```
 
+#### `addClientDirective` option
+
+<span style="margin: 0.25em;"><Badge variant="accent">Experimental</Badge></span>
```

Comment:
I'm not really sure how we represent experimental options here, so I tried a Badge to show it. The screenshot in the PR description should show how it looks like.

Reply:
We'll know for sure once the Netlify deploy preview runs!  :sweat_smile: